langchain4j-azure-open-ai/src/test/java/dev/langchain4j/model/azure/AzureOpenAiStreamingChatModelIT.java

```diff
@@ -61,6 +61,8 @@ void should_stream_answer(String deploymentName, String gptVersion, boolean useA
                 .logRequestsAndResponses(true)
                 .build();
 
+        Thread.sleep(60_000);
```

Comment:
minor: it would be much easier to do it in `@AfterEach` instead of adding `Thread.sleep()` in each test:
```java
    @AfterEach
    void afterEach() throws InterruptedException {
        Thread.sleep(10_000L); // to avoid hitting rate limits
    }
```

BTW, isn't 60 seconds too much?

Reply:
The error messages I got was telling me to wait 50 seconds, that’s why I put 60.

Reply:
That is strange, I've jsut run this test class without `Thread.sleep` and it works fine.

But I have stumbled into another problem. Might be a bug on Azure side?

`should_use_json_format` fails with the following error:
```
  "error": {
    "message": "'messages' must contain the word 'json' in some form, to use 'response_format' of type 'json_object'.",
    "type": "invalid_request_error",
    "param": "messages",
    "code": null
  }
```

Message definitely contains json: `Return JSON with two fields: name and surname of Klaus Heisler.`

I suspect it might be because the message text is base64 encoded?
`2024/06/13 10:54:07,971 1342 [INFO  ] [main] implementation.OpenAIClientImpl$OpenAIClientService.getChatCompletions - {"az.sdk.message":"HTTP request","method":"POST","url":"https://langchain4j-test-east-us.openai.azure.com//openai/deployments/gpt-4o/chat/completions?api-version=2024-05-01-preview","tryCount":1,"Date":"Thu, 13 Jun 2024 08:54:07 GMT","Content-Type":"application/json","x-ms-client-request-id":"be95b4c9-cbc5-4383-bd58-7c09e535853e","accept":"application/json","User-Agent":"langchain4j-azure-openai","redactedHeaders":"api-key","content-length":233,"body":"{\"messages\":[{\"content\":\"UmV0dXJuIGpzb24gd2l0aCB0d28gZmllbGRzOiBuYW1lIGFuZCBzdXJuYW1lIG9mIEtsYXVzIEhlaXNsZXIu\",\"role\":\"user\"}],\"max_tokens\":50,\"temperature\":0.0,\"stream\":true,\"model\":\"gpt-4o\",\"response_format\":{\"type\":\"json_object\"}}"}`